app/models/notification.rb

```diff
@@ -1,15 +1,21 @@
 # frozen_string_literal: true
 class Notification < ApplicationRecord
-  include PgSearch
-  pg_search_scope :search_by_subject_title,
-                  against: :subject_title,
-                  using: {
-                    tsearch: {
-                      prefix: true,
-                      negation: true,
-                      dictionary: "english"
+  if ActiveRecord::Base.connection.adapter_name == 'PostgreSQL'
```

Comment:
I believe this should cause a DB connection to be established on application load (with the class) instead of lazily on the first query as one might expect with ActiveRecord normally.

* That doesn't really matter for us, so I guess this is fine ¯\\\_(ツ)_/¯
* On the other hand, why not just use the nice `DatabaseConfig` you just built instead? 😄